crates/ruma-events/src/enums.rs

```diff
@@ -94,6 +98,15 @@ event_enum! {
     ]
 }
 
+event_enum! {
+    /// Any basic to-device event.
+    kind: BasicToDevice,
```

Comment:
I see, so you created a new enum here. But I don't see it being used, so I'd say we don't need it.